docs/rtd/postmortems.rst

```diff
@@ -0,0 +1,5 @@
+===========
```

Comment:
I don't think this should belong to RTD documentation.

Reply:
Do you mean that we should put the Postmortems to the [github wiki](https://github.com/avast/yaramod/wiki)?

Reply:
I'm not even sure we need it for yaramod.